addons/sale/models/sale_order_line.py

```diff
@@ -158,6 +158,10 @@ def _compute_tax_id(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get('display_type'):
```

Comment:
It doesn't cover the case where the line is created with `default_display_type` in the context, but I don't think it happens (at least in our codebase).